impeller/renderer/backend/gles/buffer_bindings_gles.h

```diff
@@ -57,6 +57,7 @@ class BufferBindingsGLES {
   std::vector<VertexAttribPointer> vertex_attrib_arrays_;
 
   std::unordered_map<std::string, GLint> uniform_locations_;
+  std::vector<uint8_t> array_element_buffer_;
```

Comment:
Why are we adding this as an instance variable?  I'd rather keep it localized to where it is used to avoid stale data bugs.

Reply:
I was trying to avoid creating the empty vector, since I can't safely conditionally initialize it. But that is probably not a worthwhile change

Reply:
Removed